Userland/Libraries/LibJS/Tests/builtins/Intl/Collator/Collator.prototype.compare.js

```diff
@@ -0,0 +1,108 @@
+describe("correct behavior", () => {
```

Comment:
Seems like something went weird with squashing? This is a copy-paste of the `resolvedOptions` test

Reply:
Not sure how that happened :thinking:, fixed